.golangci.yml

```diff
@@ -8,3 +8,8 @@ linters:
   enable:
     - gofumpt
     - errorlint
+    - revive
+
+issues:
+  include:
+    - EXC0002
```

Comment:
Is this a warning produced by golint? Since this commit does not enable golint, it makes no sense.

Or, if it is also produced by revive, I think a comment is needed here, telling what this does.